src/types/peer.rs

```diff
@@ -0,0 +1,66 @@
+use ethereum_types::U256;
+use serde_derive::{Deserialize, Serialize};
+
+use std::fmt;
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct PeerType {
+    pub active: usize,
+    pub connected: usize,
+    pub max: u32,
+    pub peers: Vec<PeerInfo>,
+}
+
+impl fmt::Debug for PeerType {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "active: {}, connected: {}, max: {}, peers: {:?}", self.active, self.connected, self.max, self.peers)
+
+    }
+}
+
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct PeerInfo {
+    pub id: Option<String>,
+    pub name: String,
+    pub caps: Vec<String>,
+    pub network: PeerNetworkInfo,
+    pub protocols: PeerProtocolsInfo,
+}
+impl fmt::Debug for PeerInfo {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "id: {:?}, name: {}, caps: {:?}, remote address: {}, local address: {}", self.id, self.name, self.caps, self.network.remote_address, self.network.local_address)
+    }
+}
+
+
+#[derive(Serialize, Clone, Deserialize, Debug)]
+pub struct PeerNetworkInfo {
+
+    #[serde(rename="remoteAddress")]
```

Comment:
could use `#[serde(rename_all = "camelCase")]` on the entire struct

Reply:
done.